Tests/ApolloCodegenTests/TestHelpers/LineByLineComparison.swift

```diff
@@ -1,26 +1,46 @@
 import Foundation
 import Nimble
+import ApolloTestSupport
 import ApolloCodegenTestSupport
 
 /// A Nimble matcher that compares two strings line-by-line.
-public func equalLineByLine(_ expectedValue: String) -> Predicate<String> {
+///
+/// - Parameters:
+///   - expectedValue: The expected string to match against
+///   - atLine: [optional] The line in the actual value where matching should begin.
+///   If provided, the actual value will be compared to the lines at the given range.
+///   Defaults to `nil`.
+public func equalLineByLine(
+  _ expectedValue: String,
+  atLine startLine: Int = 1,
```

Comment:
probably worth specifying that this is 1-indexed and not zero-indexed since the array of lines is in fact zero-indexed

Reply:
also this is a genius idea

Reply:
Fair point, I'll add documentation to this in my next PR.

And thank you! :)

Reply:
FWIW, the whole matcher is based off of your `LineByLine` comparison that already existed, just refactored it into a nimble matcher and then added the `atLine` functionality!